contrib/git-lfs/template.py

```diff
@@ -0,0 +1,36 @@
+pkgname = "git-lfs"
+pkgver = "3.5.1"
+pkgrel = 0
+build_style = "go"
+make_cmd = "gmake"
+make_build_args = [
+    "-ldflags",
+    "-X github.com/git-lfs/git-lfs/v3/config.Vendor=ChimeraLinux",
+]
+hostmakedepends = ["asciidoctor", "go"]
+checkdepends = ["bash", "git", "gmake"]
```

Comment:
tests pass for me without gmake
```suggestion
checkdepends = ["bash", "git"]
```

Reply:
actually, by using the go build style for this we're missing out on running some tests (at least integration tests), maybe we should make the whole template use the make build style instead

Reply:
Oh! I'll look into it.

Reply:
Actually I'd initially made the template using make build style.

But the included makefile is a mess IMHO, so I just reverted to building it using the go build style and doing the other bits myself.

Reply:
_Latest commit needs gmake to run tests._